src/main/java/Task.java

```diff
@@ -0,0 +1,35 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public boolean isDone() {
+        return isDone;
+    }
+
+    public void markAsNotDone() {
+        this.isDone = false;
+    }
+
+    public String toString() {
+        return "[ ][" + getStatusIcon() + "] " + getDescription(); // Adds a generic task identifier [G]
```

Comment:
I like the use of different methods to fetch the different strings such as the icon and the description !